packages/tools/src/stateManagement/segmentation/removeSegmentationsFromToolGroup.ts

```diff
@@ -55,13 +53,14 @@ function removeSegmentationsFromToolGroup(
   }
 
   segRepresentationUIDsToRemove.forEach((segmentationDataUID) => {
-    _removeSegmentation(toolGroupId, segmentationDataUID);
+    _removeSegmentation(toolGroupId, segmentationDataUID, immediate);
```

Comment:
Why would you not want to do it immediately? What does this mean?

Reply:
If one is removing couple of 10 segmentations and don't want to render after each?